src/modules/style.ts

```diff
@@ -1,6 +1,11 @@
 import {VNode, VNodeData} from '../vnode';
 import {Module} from './module';
 
+export type VNodeStyle = Record<string, string> & {
+  delayed: Record<string, string>
+  remove: Record<string, string>
+}
```

Comment:
This is an improvement over the last commit, because it is slightly more specific, in that only `delayed` and `remove` can have `Record`s as values.